product_unique_serial/wizard/stock_serial.py

```diff
@@ -156,47 +174,18 @@ class StockSerialLine(models.TransientModel):
 
     _name = 'stock.serial.line'
 
+    @api.multi
+    @api.depends('serial')
+    def _compute_lot_id(self):
```

Comment:
but you are not considering all cases of the onchange like return
 you are validating all cases when press button of the wizard and then if there are many serial captured and there is warning/exception in one of the serial the user does not know until end of process